config/environments/development.rb

```diff
@@ -50,6 +50,15 @@
     path: "/:class/:attachment/:id_partition/:style/:filename"
   }
 
+  config.action_mailer.delivery_method = :sendmail
+  config.action_mailer.sendmail_settings = {
+    location: ENV['SENDMAIL_PATH'],
+    arguments: ''
+  }
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.default_options = {from: 'no-reply@flashcards.local'}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.